### PR TITLE
Align with latest version of webrtc-stats hierarchy

### PR DESCRIPTION
**Pull request by dontcallmedom**
Removed stats for stream, track (now obsolete)
Added media-source, transceiver, sender, receiver, ice-server stats
Aligned fields of various stat objects with new hierarchy
Aligned list of MTI stats with latest WebRTC spec

### CODE REVIEW

**Comment by dontcallmedom** [comment]
@jan-ivar any chance you could review this?

File: webrtc/RTCStats-helper.js [hunk 1]
```diff
@@ -145,42 +149,29 @@ function validateRtcStats(statsReport, stats) {
   [webrtc-stats]
   7.1.  RTCRTPStreamStats dictionary
     dictionary RTCRTPStreamStats : RTCStats {
```
Comment (henbos) [inline webrtc/RTCStats-helper.js:151]: nit: RTCRTPStreamStats has been renamed RTCRtpStreamStats and the same with RTCRececeivedRtpStreamStats, RTCSentRtpStreamStats, RTCOutboundRtpStreamStats, RTCInboundRtpStreamStats, RTCRemoteOutboundRtpStreamStats and RTCRemoteInboundRtpStreamStats. You may optionally update the casing in this PR to reflect that.

File: webrtc/RTCStats-helper.js [hunk 2]
```diff
@@ -662,41 +864,33 @@ function validateTransportStats(statsReport, stats) {
                           'transport');
 
   assert_optional_enum_field(stats, 'iceRole',
-    ['controlling', 'controlled']);
+                             ['unkwnon', 'controlling', 'controlled']);
```
Comment (henbos) [inline webrtc/RTCStats-helper.js:867]: typo: 'unknown'

**Review by henbos — APPROVED** [review]
LGTM. I didn't find any mistakes, except one typo

**Comment by dontcallmedom** [comment]
@foolip can you help me figure out what the CI error is?

**Comment by foolip** [comment]
@dontcallmedom I think you've run into https://github.com/web-platform-tests/wpt/issues/23312. There's also a conflict, however, so if you resolve that conflict chances are that the problem won't happen again. (To be clear, I don't think having a conflict increases the chance of it happening.)

**Pull request merged**